npm script flow.compile:system not working on windows

npm run flow.compil.system on windows causes following issue

```
> ng2-formly@2.0.0-alpha.5 flow.compile:system C:\Users\divyjain\Documents\GitHub\ng2-formly
> ./.config/bundle-system.js

'.' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "flow.compile:system"
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! ng2-formly@2.0.0-alpha.5 flow.compile:system: `./.config/bundle-system.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ng2-formly@2.0.0-alpha.5 flow.compile:system script './.config/bundle-system.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ng2-formly package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./.config/bundle-system.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ng2-formly
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ng2-formly
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\divyjain\Documents\GitHub\ng2-formly\npm-debug.log
```

To fix this problem created bundle-system and bundle-system.cmd in .config folder and update npm script location but it failed with similar error. But surprisingly it works when i move bundle-system.cmd to node_module/.bin

--- a/package.json
+++ b/package.json
@@ -10,9 +10,9 @@
     "flow.install:typings": "./node_modules/.bin/typings install",
     "flow.compile": "npm run flow.install:typings && npm run flow.compile:common && npm run flow.compile:system ",
     "flow.compile:common": "./node_modules/.bin/tsc",
-    "flow.compile:system": "./.config/bundle-system.js",
+    "flow.compile:system": "node ./.config/bundle-system.js",
     "flow.clean": "./node_modules/.bin/del dist \"dist/components\" \"dist/services\" \"dist/templates\"",
-    "commit": "git-cz",
+    "commit": "npm run flow.compile && git-cz",
     "prepublish": "npm run flow.clean && npm run flow.compile"
   },
   "keywords": [

